api/v1alpha1/database_types.go

```diff
@@ -233,9 +233,17 @@ type SLAResponse struct {
 
 // +kubebuilder:object:generate:=false
 type DatabaseNode struct {
-	Id               string `json:"id"`
-	Name             string `json:"name"`
-	DatabaseServerId string `json:"dbServerId"`
+	Id               string         `json:"id"`
+	Name             string         `json:"name"`
+	DatabaseServerId string         `json:"dbServerId"`
+	DbServer         DatabaseServer `json:"dbserver"`
```

Comment:
if we support clustered DBs, will we store/show a list of dbservers? if so, then this has to be a list otherwise it will break api compatibility when we move to HA systems